includes/frontend.php

```diff
@@ -238,7 +238,7 @@ public function init() {
 		$document = Plugin::$instance->documents->get( $this->post_id );
 
 		if ( is_singular() && $document && $document->is_built_with_elementor() ) {
-			add_action( 'wp_enqueue_scripts', [ $this, 'enqueue_styles' ] );
+			add_action( 'wp_enqueue_scripts', [ $this, 'enqueue_styles' ], 20 );
```

Comment:
Can you add a const for this `20`? So that we don't have to guess what it's about :)
`AFTER_SOMETHING`

Reply:
Requested by @KingYes, after a code review.

This way when you register styles in widgets, you can use the default 10 value.

To do that, when styles are enqueue, they should be above the default 10.

We want this value to be hardcoded.

Reply:
@rami-elementor Can you add a const or comment? Otherwise another developer would need to guess what `20` means.